sdk/core/System.ClientModel/tests/System.ClientModel.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <IsTestProject>true</IsTestProject>
     <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);SCM0001</NoWarn>
```

Comment:
Is this permanent?

Reply:
that one is permanent, in this test project we want to force a few hand written context classes vs splitting into many test projects.